examples/http-certification/assets/dfx.json

```diff
@@ -6,12 +6,12 @@
     "backend": {
       "type": "custom",
       "candid": "src/backend/backend.did",
-      "wasm": "../../../target/wasm32-unknown-unknown/release/http_certification_new_assets_backend.wasm",
```

Comment:
`http_certification_new_assets_backend` was meant to be a temp name that I forgot to fix